Fitness/Controllers/GymDetailViewController.swift

```diff
@@ -17,6 +17,12 @@ struct HoursData {
     var isDropped: Bool!
 }
 
+struct FacilityData {
+    var facility: String!
```

Comment:
Do these need to be var? Can each of these be `let name: Type` without the `!` either?